docarray/document/data.py

```diff
@@ -111,3 +113,25 @@ def _set_default_value_if_none(self, key):
                     setattr(self, key, defaultdict(NamedScore))
                 else:
                     setattr(self, key, v() if callable(v) else v)
+
+    @staticmethod
+    def _embedding_eq(array1: 'ArrayType', array2: 'ArrayType'):
```

Comment:
I guess a similar method needs to be implemented for tensor

Reply:
you are right, added